qa/rpc-tests/test_framework/util.py

```diff
@@ -283,7 +314,11 @@ def initialize_chain(test_dir,bitcoinConfDict=None,wallets=None, bins=None):
         wait_bitcoinds()
         disable_mocktime()
         for i in range(4):
-            os.remove(log_filename("cache", i, "debug.log"))
+            if bins:
```

Comment:
I propose that during node object initialization add a member variable: bins[i].logFilename
so we do not need to write flowee exceptional code everywhere.  This code then becomes:
```
os.remove(bins[i].logFilenema)
```